auth-service/README.md

```diff
@@ -12,11 +12,7 @@
 
```

Comment:
Should line 10 recommend `docker compose up -d postgres`?

Reply:
The app in docker compose is has a profile set, so `docker compose up` will not start it, it would have to be `docker compose --profile app up`. Could change it just for clarity, but it should work as-is.